config/environments/development.rb

```diff
@@ -13,19 +13,23 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
+  # Run rails dev:cache to toggle caching.
+  if Rails.root.join('tmp', 'caching-dev.txt').exist?
```

Comment:
Nul besoin de séparer les arguments ici, un `/` fonctionne très bien partout.

```suggestion
  if Rails.root.join('tmp/caching-dev.txt').exist?
```